src/lib/archwizard.module.ts

```diff
@@ -15,6 +15,13 @@ import {WizardCompletionStepDirective} from './directives/wizard-completion-step
 import {WizardStepSymbolDirective} from './directives/wizard-step-symbol.directive';
 import {WizardStepTitleDirective} from './directives/wizard-step-title.directive';
 import {WizardStepDirective} from './directives/wizard-step.directive';
+import {NAVIGATION_MODE_FACTORY, NavigationModeFactory} from './navigation/navigation-mode-factory.interface';
+import {BaseNavigationModeFactory} from './navigation/navigation-mode-factory.provider';
+
+
+export interface ArchwizardModuleConfig {
```

Comment:
Please add some comments to the interface and its method(s)

Reply:
Done in 01b3dd7